jupyterhub/files/hub/jupyterhub_config.py

```diff
@@ -12,6 +12,22 @@
 
 from z2jh import get_config, set_config_if_not_none
 
+def serviceUrl(svc, ssl=False):
+    port = int(os.environ['{}_SERVICE_PORT'.fornat(svc.upper())])
```

Comment:
```suggestion
    port = int(os.environ['{}_SERVICE_PORT'.format(svc.upper())])
```
Travis is failing due to this error